README.md

```diff
@@ -81,6 +81,27 @@ puts response.status_code
 puts response.body
 puts response.headers
 ```
+## Manage Account Settings
```

Comment:
The current links are in-line like so: 
- https://github.com/sendgrid/sendgrid-ruby/blob/master/examples/mail/mail.rb#L29
- https://github.com/sendgrid/sendgrid-ruby/blob/master/USAGE.md#v3-mail-send

But perhaps we can add a helpers section in the README too.